src/chi/components/steps/steps.scss

```diff
@@ -87,5 +92,131 @@
         color: $brand-color;
       }
     }
+
+    &.-horizontalLabels {
+      background-color: set-color(white);
+      counter-reset: step;
+      padding: 1rem;
+
+      & > li {
+        &::before {
+          background-color: set-color(white);
+          border: 0.125rem solid $brand-color;
+          content: counter(step);
+          counter-increment: step;
+          font-size: 0.8125rem;
+          height: 1.3rem;
+          margin: 0 auto;
+          width: 1.3rem;
+        }
+
+        &::after {
+          top: 0.5625rem;
+          width: 90%;
+        }
+
+        &.-active {
+          &::before {
+            box-shadow: 0 0 0 0.125rem set-color(white);
+          }
+
+          & ~ li {
+            &::before {
+              background-color: set-color(white);
+              border-color: set-color(grey, 40);
+            }
+
+            & a {
+              color: set-color($brand-color);
+            }
+          }
+
+          & a {
+            margin-left: 0.75rem;
+          }
+        }
+
+        &.-completed {
+          &::before {
+            border: .125rem solid set-color($brand-color);
+            content: "\e91b";
+            font-family: 'chi' !important;
+            font-size: 0.75rem;
+          }
+        }
+      }
+
+      & a {
+        background-color: set-color(white);
+        color: $brand-color;
+        margin-left: 0.65rem;
+        padding-left: 0.5rem;
+        padding-right: 0.5rem;
+        position: absolute;
+        top: 0.125rem;
+      }
+    }
+
+    &.-inverse {
+      background-color: set-color(black);
+      padding: 1rem;
```

Comment:
@davidojedacl this padding can be removed as it is duplicated from .-horizontalLabels